packages/odyssey-react-mui/src/Radio.tsx

```diff
@@ -59,7 +63,23 @@ const Radio = ({
   value,
   onChange: onChangeProp,
   onBlur: onBlurProp,
+  inputRef,
 }: RadioProps) => {
+  const ref = useRef(null);
+
+  useImperativeHandle(
+    inputRef,
+    () => {
+      const element = ref.current as unknown as HTMLElement;
+      if (element.tagName === "INPUT") {
+        return element as HTMLInputElement;
+      }
+      const inputElement = element.querySelector("input");
+      return inputElement as HTMLInputElement;
+    },
```

Comment:
Haven't used `useImperativeHandle` hook before, doesn't this expose all methods from HTMLInputElement? You probably want to limit it to the methods we need, e.g., `focus()`